examples/jenkins/README.md

```diff
@@ -10,46 +9,80 @@ the deployment works, and then tag the test version into production.
+            Labels:     <none>
+            Annotations:    kubernetes.io/service-account.name=default,kubernetes.io/service-account.uid=32f5b661-2a8f-11e5-9528-3c970e3bf0b7
+            Type:   kubernetes.io/service-account-token
+            Data
+            ====
+            ca.crt: 1066 bytes
+            token:  eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJ0ZXN0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImRlZmF1bHQtdG9rZW4taXp2MXUiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiZGVmYXVsdCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjMyZjViNjYxLTJhOGYtMTFlNS05NTI4LTNjOTcwZTNiZjBiNyIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDp0ZXN0OmRlZmF1bHQifQ.sw0Sr8dUj4DNe49tIfGYXYAyCSpmIs6c1YIBpBMXtgdi9YBEJnV6aDc2k_D8kEddtRdpgxNwcwiey2R7lnJxRElWDApO86PMdMW9CDJXHgU7SjJ7x2H_pXFZjncHK4_dzE-juHJgmamAHW_8An9fYa8fn9TBxoLR-kAQYqxPZXD_fApGBxk2XZRoVI5jKEgRAWT5e3O_PWwlbKuUu2siakPQaziQ1YCD2LhCwI7wThDE01EaqlRNxBFOk1roOs4oKAO2g0xyaAHzYOwrXakvJGAhJHY_MpvTK2y5kGaKvmdmeYk49SGx5dC_JafIc44HntnNXq6jZax_9NBrF78wRA
 
-6. Add API credentials to the Jenkins build:
+        Copy the value from the `token` field, it will be used later.
 
-    1. Go back to your browser, refresh and select the rubyJob build job.
-    2. Choose `Configure`.
-    3. Locate the KUBECONFIG_CREDENTIALS parameter, and replace the default value with the contents of the `openshift.local.config/master/openshift-client.kubeconfig` file.
-    4. Click `Save`.
+4. Create and deploy the Jenkins service
 
-6. Run the Jenkins build
+        $ oc new-app jenkins-ephemeral-template.json
+    
+    Note the ip and port of the Jenkins service reported by this command, you will need it later.
+
+        $ oc deploy jenkins --latest
+
+    **Note**: This template uses an EmptyDir type volume.  If you want to ensure your jenkins configuration/job information is persisted through pod restarts and deployments, you will want to use a [PersistentVolume](https://docs.openshift.org/latest/admin_guide/persistent_storage_nfs.html) instead.
+    
+5. Create the sample application configuration
+
+        $ oc new-app application-template.json
+ 
+6. Open the Jenkins service ip:port from step 4 in your browser.  Once it is available, login using username `admin` and password `password`.
    
-    1. In the browser, select the rubyJob build job and choose `Build with parameters`.
-    2. Click `Build` (you should not need to modify any parameters).
+7. Select the the `OpenShift Sample` job and click `Configure`
+
+8. Optional (if the default value does not resolve to your OpenShift api endpoint): Fill in the `OPENSHIFT_API_URL` value with `https://hostname:port` where hostname/ip and port are for your OpenShift api endpoint.
+
+9. Optional (if not running Jenkins in your same project): Fill in the `AUTH_TOKEN` value with the token value retrieved in step 3.
```

Comment:
updated the text about cross-cluster but i'm going to punt on describing how to grant cross-project permissions.  trying to keep this simple-ish.